src/state/CallViewModel/localMember/LocalMember.ts

```diff
@@ -238,16 +268,16 @@ export const createLocalMembership$ = ({
   /**
    * Extract the tracks from the published. Also reacts to changing publishers.
    */
-  const tracks$ = scope.behavior(
-    publisher$.pipe(switchMap((p) => (p?.tracks$ ? p.tracks$ : constant([])))),
-  );
-  const publishing$ = scope.behavior(
-    publisher$.pipe(switchMap((p) => p?.publishing$ ?? constant(false))),
-  );
-
-  const startTracks = (): Behavior<LocalTrack[]> => {
+  // const tracks$ = scope.behavior(
+  //   publisher$.pipe(switchMap((p) => (p?.tracks$ ? p.tracks$ : constant([])))),
+  // );
+  // const publishing$ = scope.behavior(
+  //   publisher$.pipe(switchMap((p) => p?.publishing$ ?? constant(false))),
+  // );
+
+  const startTracks = (): Behavior<void> => {
     trackStartRequested.resolve();
-    return tracks$;
+    return constant(undefined);
   };
```

Comment:
Yes lets remove the return type entirely or add a comment that this should return `tracks$`